src/main/resources/site/content-types/article/article.ts

```diff
@@ -30,9 +30,59 @@ export interface Article {
   }>;
 
   /**
-   * Løpenummer
+   * Relatert artikkel
```

Comment:
This change shows up in #304 too ! The change was actually made as part of #284, i think ? Not sure how/why it pops up in other branches/PRs like this !

@Garlov : any thoughts ?

Reply:
I'm guessing it's because of a merge 🤷‍♂️